src/components/MainComponents/Intro.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable */
+
+import React from 'react'
+import styled from 'styled-components'
+
+
+export const Intro = () => {
+    return (
+        <div> 
+            <Text> 
+            <Line></Line>
+            <Dot></Dot>
+            <> </>
+               With more than eight years of experience in finance I’m now deep diving into code and trying out a new Path. =IF only coding was as simple as Excel, but I’m all about solving problems and creating  #VALUE! 
```

Comment:
Ha ha this is great! Lot's of personality shining through - exactly what you want in a portfolio